core/templates/dev/head/css/oppia.css

```diff
@@ -2614,6 +2628,20 @@ div#ng-curtain {
   margin: auto;
 }
 
+.form-control.oppia-rte-content > div > p {
```

Comment:
Can you combine these with the previous set of rules (by separating them with a comma)? Don't want them to get out of sync -- if you put them in totally separate parts of the file, it's very likely that, in the future, someone will change one of them and not the other.